### PR TITLE
Alterations to blood loss effect

### PR DESCRIPTION
**Pull request by chaosvolt**
This is...very fucking likely to break spectacularly. Currently attempting to implement blood loss as a fatal effect via addition of a body part for it, which the blood loss effect damaging that part instead
of the bleeding body part.

Will need to see what else will be required to work past the inevitable commit failures.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Hmm. And now the endless waiting to see if it'll be failed or not. Weird, bot's being slow I guess.

Though I'd be willing to bet it'll be a fail.

**Comment by chaosvolt** [comment]
There we go. It was inevitable. Now to see what I fucked up.

File: src/bodypart.h [hunk 1]
```diff
@@ -18,6 +18,7 @@ enum body_part : int {
     bp_leg_r,
     bp_foot_l,
     bp_foot_r,
+    bp_blood,
```
Comment (Coolthulhu) [inline src/bodypart.h]: Blood as a body part is more trouble than it's worth.

You could implement it as a separate value, like oxygen.

**Comment by chaosvolt** [comment]
Hmm. Well...scheisse. We'd need to rework the bleed effect and such though, right?

**Comment by Coolthulhu** [comment]
You could just hardcode it to remove blood, in `player::hardcoded_effects`.

Believe me, it is worth it in the long run, otherwise you'll be catching cases like "frostbite (blood)", "bite (blood)", wet blood and "hits you in the blood" for a while.

**Comment by chaosvolt** [comment]
Hmm. If blood can also be set to restore over time via the same mechanism as natural healing? Will also have to see how to change the bleeding effect from damaging a given part to reducing the value directly.

**Comment by chaosvolt** [comment]
Well. Github client is currently frozen up staring at an uncommited change (that I didn't do, seeing as it seems to be claiming an alteration to the Polish translation file) for a branch that was merged ages ago. Oh hey, did you know it's a few good hours into monday over here? I sure do.

At this point I'm sorely tempted to close this PR, and leave the work to someone else.

**Pull request closed without merging**